monitor/puller.go

```diff
@@ -14,7 +17,7 @@ import (
 )
 
 func (m *Monitor) processPullerQueue() {
-	if !m.isPuller() {
+	if !m.isSyncer() {
```

Comment:
rename the file?

Reply:
We combine puller and pusher to syncer, this file is still about the puller functions